wagtail/wagtailadmin/tests/test_jinja2.py

```diff
@@ -11,11 +7,9 @@
 from wagtail.wagtailcore.models import Page, PAGE_TEMPLATE_VAR, Site
 
 
-@unittest.skipIf(django.VERSION < (1, 8), 'Multiple engines only supported in Django>=1.8')
 class TestCoreJinja(TestCase):
 
     def setUp(self):
-        # This does not exist on Django<1.8
         from django.template import engines
```

Comment:
This could be moved to a top level import now that it will not throw ImportErrors for old Django versions